ui/src/app/utils/index.tsx

```diff
@@ -630,3 +630,20 @@ export const useId = () => {
   const [id] = useState(() => fp.uniqueId('element-'));
   return id;
 };
+
+const nothing = {};
+
+// maybe - takes a function and a value. If the value is not defined returns "nothing"
+// Feel free to export this when a proper use case arises
```

Comment:
I love this note. I sometimes wonder if the author had a good reason to *not* export something.

Reply:
Added comment